### ISSUE TITLE
A11y_Azure Tools Storage Explorer_SAS URL_High Contrast: In High Contrast white mode keyboard focus is not visible on the controls in the Microsoft Azure Storage Explorer popup

### ISSUE THREAD

**Issue by v-kanve** [microsoft/AzureStorageExplorer#4990]
**GitHubTags**

#A11yTCS;#A11ySev2;#A11yMAS;#DesktopApp;#Azure Tools Storage Explorer;#BM-AzureToolsStorageExplorer-Win32-Sep2021;#WCAG1.4.3;#Benchmark;Win32;#High Contrast;

**Environment Details**
Application Name: Azure Tools Storage Explorer
Windows Version: Windows 10

**Repro Steps**
1.Apply High Contrast White theme
2.Open Azure Tools Storage Explorer
2.Tab till the "Blob containers" and press Enter.
3.Click on any existing blob container.
4.Navigate to "Upload" button and Upload a blob/file by using "Upload Files" control.
5.Verify whether keyboard focus is visible on all the controls in the pop or not

**Comment by MRayermannMSFT** [comment]
Important note: this is only repro'ing if Windows is also in high contrast white theme. When windows is in normal theme, the coloring is way better:
![image](https://user-images.githubusercontent.com/21112697/135546147-b6ecb3a1-5f37-41dd-9813-8d6025848c8d.png)
![image](https://user-images.githubusercontent.com/21112697/135546166-6a02f1ae-c161-4519-943f-b10eb3033e4c.png)

**Comment by MRayermannMSFT** [comment]
Related: #4636

**Comment by v-inanda** [comment]
Regressed the bug in the Version :1.22.0-dev
AzCopy version: 10.12.2
The issue is fixed so closing the bug. Please find the attachment for reference.
![4990_Fix](https://user-images.githubusercontent.com/91525824/138220964-a93d6bec-7066-4a60-8e2b-a5110e82be1f.gif)

**Comment by Sobi261** [comment]
#Closed;
